docs/releases/5.0.txt

```diff
@@ -551,6 +551,9 @@ Miscellaneous
   content area are now rendered in a ``<main>`` and ``<header>`` tag instead of
   ``<div>``.
 
+* In order to improve accessibility, the admin's changelist filter is now
+  using a ``<nav>`` landmark instead of a ``<div>``.
```

Comment:
Move it to the `5.1.txt`.

Reply:
Moved to `5.1.txt` as per `[2daca5e](https://github.com/django/django/pull/17309/commits/2daca5e187015effa39d9ca6167ee26e4fb64296)`